echidna-lb/src/dispatcher/algorithms/ip_hashing.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{dispatcher::Dispatcher, backend::Backend};
+use actix_web::HttpRequest;
+use std::collections::hash_map::DefaultHasher;
+use std::hash::Hash;
+use std::hash::Hasher;
+use std::sync::atomic::Ordering;
```

Comment:
nit: backend.rs has a contradicting import sequence, i.e. local modules are listed after external crates. This order here seems more reasonable though.

Reply:
Fixed in all the places.